Install.md

```diff
@@ -87,19 +68,25 @@ This guide helps you build and install Snort.
     ```sh
     cd snort-2.9.8.3/
     ```
-    
-2. Run the configuration script.
+2. Prepare for automake, then autocreate makefile. 
     ```sh
-    ./configure --enable-sourcefire
+    aclocal
+    autoconf
+    autoheader
+    automake -a
+    ```
+3. Run the configuration script.
+    ```sh
+    ./configure --enable-sourcefire --enable-static --disable-shared
     ```
 
-3. Create Patch of Makefille      
+4. Create Patch of Makefille. This part is critical for the next steps. The defaults are blank. Please type in the answers manually.
    ```sh
    cd ../
    ./patching-Makefile.sh
    ```
 
-4. Navigate to the src folder of snort and Make snort.
+5. Navigate to the src folder of snort and Make snort.
```

Comment:
Please add a step asking the user to run `autoreconf -ivf` in case of a version mismatch error, then to do step 3 again. 
Also, I am getting this issue: 
<img width="1280" alt="Screen Shot 2019-10-24 at 11 57 43 PM" src="https://user-images.githubusercontent.com/32916582/67542278-2369d880-f6ba-11e9-80b7-dcceb6cb1cf6.png">

Could you elaborate on -ldpdk a bit more?

Reply:
Yes, looks like I forgot to add `autoreconf -fvi`. Thanks for pointing that out!

Reply:
Hi Archit, please add this step to the install guide. Thank you.